site/pages/home/blog-slice/blog-entry/index.js

```diff
@@ -1,30 +1,43 @@
-import React, { PropTypes } from 'react';
+// @flow
+
+import React from 'react';
 import styles from './style.css';
 
-const BlogEntry = ({
-  category,
-  title,
-  url,
-  authorName,
-  authorTitle,
-}) => {
+type Author = {|
+  bio: string,
+  displayName: string,
+|}
+
+export type BlogPost = {|
+  urlId: string,
+  categories: Array<string>,
+  title: string,
+  author: Author,
+|}
+
+const matchAuthorTitle = title => {
+  const match = title.match(/<.+>(.*)<.+>/);
+  return (match && match[1]) || title;
```

Comment:
Oh Squarespace. Can we do this in the fetcher maybe? The less data in state, the better.